timeless/restaurants/table_shapes/views.py

```diff
@@ -1,45 +1,53 @@
-"""Floors views module.
-@todo #72:30min Continue implementing list(), create(), edit() and
- delete() methods, using SQLAlchemy and TableShape model. In the index page it
- should be possible to sort and filter for every column. Table shape management
- page should be accessed by the Location page. Update html templates when
- methods are implemented. Create more tests for all methods.
+"""TableShape views module.
+ @todo #162:30min Update html templates when all
+ methods are implemented.
 """
 from flask import (
     Blueprint, flash, redirect, render_template, request, url_for
 )
 
+from timeless.db import DB
+from timeless.restaurants import models
+from timeless.restaurants.table_shapes import forms
+
+
 bp = Blueprint("table_shape", __name__, url_prefix="/table_shapes")
 
 
 @bp.route("/")
 def list():
-    """ List all table shapes"""
-    table_shapes = [{
-        "id": 1,
-        "description": "This is a test",
-        "picture": "shape.png"
-        }]
+    """List all table shapes
+    @todo #162:30min Implement ordering and sorting of TableShape model for
+    all columns. Do not forget write tests.
+    """
     return render_template(
-        "restaurants/table_shapes/list.html", table_shapes=table_shapes
-    )
+        "restaurants/table_shapes/list.html",
+        table_shapes=models.TableShape.query.all())
 
 
 @bp.route("/create", methods=("GET", "POST"))
 def create():
     """ Create new table shape"""
-    if request.method == "POST":
-        flash("Create not yet implemented")
-    action = "create"
+    form = forms.TableShapeForm(request.form)
+
+    if request.method == 'POST' and form.validate():
+        table_shape = models.TableShape()
+        form.populate_obj(table_shape)
+
+        DB.session.add(table_shape)
+        DB.session.commit()
```

Comment:
@vryazanov I wouldn't really call the DB directly from the views layer. Shouldn't these CRUD operations be abstracted behind a "controller" or "service" at least?

Reply:
@amihaiemil actually this is a common pattern in most python frameworks. I think we just need to develop smart class based view which can do CRUD operations and encapsulate the logic working with the database

Reply:
@vryazanov 
> actually this is a common pattern in most python frameworks

It seems strange to me. Not a minimum of separation of concerns? 

@emilianodellacasa WDYT here?

Reply:
@amihaiemil I agree that current implementation looks incomplete. Quite often forms are responsible for saving models to db, for example:
```
form = forms.TableShapeForm(request.form)
form.save()
```
so the form like a controller for that. I think I can leave a puzzle for implementing generic solution of that. @amihaiemil @emilianodellacasa WDYT?

Reply:
@vryazanov I agree with @amihaiemil, I don't like that much having Db logic on view.

Reply:
@emilianodellacasa I've updated PR. Please have a look. Now It looks better, I believe :)